_layouts/category_index.html

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+---
+{% if page.paginator %}
+
+  <!-- Pagination is active -->
+  {% assign paginator = page.paginator %}
+
+  {% if paginator.previous_page != nil or paginator.next_page != nil %}
+    <!-- Show page number if we have multiple pages -->
+    <h1>Category index page {{ paginator.page }} / {{ paginator.total_pages }} for &ldquo;{{ page.title }}&rdquo;</h1>
+  {% else %}
+    <!-- No page number if we only have a single page -->
+    <h1>Category index for &ldquo;{{ page.title }}&rdquo;</h1>
+  {% endif %}
```

Comment:
I personally think this would be better suited to being placed down by the pagination buttons. The title for the category page could be reworded a bit to something like "Posts tagged ...". We just want to keep the details like indices and the like out of the user's hair.

Reply:
Sure, that makes sense! Are you saying to change "Older" and "Newer" to something more like "Page X-1", "Page X+1"?

Reply:
Yeah, something you see pretty frequently is `<- Older | # | Newer ->`, where `#` is the page number and its neighbors.

Reply:
Got it. I think I made most of the changes, lmk if there's anything else to fix.

Reply:
Resolving as addressed in Slack. For a summary, we decided to place these two and the page number within a flexbox container at the foot.